phpcs.xml.dist

```diff
@@ -1,108 +1,13 @@
 <?xml version="1.0"?>
 <ruleset name="WordPress Coding Standards for Performance Plugin">
-	<description>Sniffs for WordPress plugins, with minor modifications for Performance</description>
-
-	<rule ref="PHPCompatibility"/>
-	<config name="testVersion" value="7.0-"/>
-
-	<rule ref="WordPress-Docs"/>
-	<rule ref="WordPress-Extra" />
-	<rule ref="WordPress.WP.I18n"/>
-	<config name="text_domain" value="performance-lab,default,auto-sizes"/>
-
-	<arg value="ps"/>
-	<arg name="extensions" value="php"/>
+	<rule ref="phpcs.ruleset.xml"/>
+	
+	<config name="text_domain" value="performance-lab,default"/>
```

Comment:
Why are we even allowing the default text domain?

Reply:
Yep, and if we want to intentionally re-use core strings, we can add ignore comments with explanations.

Reply:
Thanks for raising this. We use the `default` text domain in two files with multiple instances. Do we need to add an inline comment to ignore these? I'm happy to do it either way.
- https://github.com/WordPress/performance/blob/trunk/admin/load.php uses it in 5 places.
- https://github.com/WordPress/performance/blob/trunk/admin/plugins.php uses it in 30 places.

Reply:
I see it's being used in multiple places within two files, so adding a comment for each line is not a good idea. Rather two points came to my mind:
- Let it be in PHPCS config, as we may add this text domain in future code.
- Try to add this on the file level instead?

Reply:
OK, I missed those two instances. In that case, could this line:

```xml
<config name="text_domain" value="performance-lab,default"/>
```

Only be applied to `admin/load.php` and `admin/plugins.php`? I don't think PHPCS has such a capability. 

In that case, I'd say we leave `default` as-is in the PL plugin's `phpcs.xml.dist` but make sure we omit it from the standalone plugins (unless we really need it). We can revisit later, as it leaving `default` would not be any different than the behavior today.

Reply:
Agreed, let's restrict usage to just these two files, either via PHPCS config specific to the files or file-level comments.

Reply:
But I don't think `config` can be selectively applied to files and I don't think comments can be used to override config values. So either we just keep `default` in the config for the PL plugin, or we add comments to disable PHPCS warnings about using `default` in those two files.

Reply:
Added exclude rules for two files in https://github.com/WordPress/performance/pull/1002/commits/5edaedf973cbcb84c953c685bac443539535dffa